api/pkg/apis/v1alpha1/managers/solution/solution-manager.go

```diff
@@ -484,6 +484,7 @@ func (s *SolutionManager) Reconcile(ctx context.Context, deployment model.Deploy
 				log.InfofCtx(ctx, " M (Solution): skipping step with role %s on target %s", step.Role, step.Target)
 				targetResult[step.Target] = 1
 				planSuccessCount++
+				summary.CurrentDeployed += len(step.Components)
```

Comment:
Why we count the currentDeployed here? Deployment hasn't started yet in this place.

Reply:
add the components for skipped step as it will move on to next step